augauto_module_path: "/home/demo/Workspace/seung/AugmentedAutoencoder"
exp_group: "exp_group/Ikea_stefan_middle"
test_config: "aae_maskrcnn_zivid.cfg"
rgb: "/zivid_camera/color/image_color"
gpu_id: "0"
width: 640
height: 400
is_thresh: 0.9
crop_offset: 0.1
padding: true
